src/cloud-api-adaptor/podvm/files/etc/systemd/system/kata-agent.service

```diff
@@ -5,6 +5,7 @@ Wants=process-user-data.service attestation-agent.service
 After=netns@podns.service process-user-data.service attestation-agent.service
 
 [Service]
+Environment=OCICRYPT_KEYPROVIDER_CONFIG=/etc/ocicrypt_config.json
```

Comment:
Will encrypted images require bumping the version of kata-agent or others in versions.yaml?

Reply:
I don't think so. it shouldn't affect kata directly, but I'll double check. there is a [bug in ocicrypt-rs](https://github.com/confidential-containers/guest-components/pull/630) which makes debugging of issues w/ enc images impractical. so we might want to bump guest-components eventually, but it shouldn't block anything in the green path atm.

Reply:
yup. I checked. it should work. this image can be used for testing:

```bash
skopeo inspect docker://ghcr.io/mkulke/nginx-encrypted:20240123 | jq -r '.Labels'
{
  "coco-key-b64": "pHSE5N+T/3GGfb/umaWgB8bfHc/dQWvmxdsjoWam0Vs=",
  "coco-key-id": "default/key/nginx-encrypted",
  "maintainer": "NGINX Docker Maintainers <docker-maint@nginx.com>"
}
```